core/src/components/datetime/datetime.tsx

```diff
@@ -1459,7 +1566,7 @@ export class Datetime implements ComponentInterface {
         class="day-column"
         color={this.color}
         items={days}
-        value={workingParts.day || this.todayParts.day}
+        value={'' + (workingParts.day || this.todayParts.day)}
```

Comment:
What is the reasoning behind this change? Did you receive an error that a number could not be assigned to `value`?

Reply:
On closer inspection, the actual reason for the error I was getting is that `DatetimeParts.day` can be `null`, but value is `string | number | undefined` (but not `null`). I'll finagle it a different way 👀